### ISSUE TITLE
Colorscale attribute descriptions in cone traces are off

### ISSUE THREAD

**Issue by jonmmease** [plotly/plotly.js#2723]
In trying out the new cone trace in plotly/plotly.py#942 I noticed that the `color` property for this trace type isn't included in the `dist/plotly-schema.json` file (at least for 1.38.2). If I understand correctly there should be a `cone.color` property alongside  `cone.colorscale`, `cone.cmin`, etc.

Here's a screenshot of my editor with all of the property bodies collapsed.

![screen shot 2018-06-11 at 3 13 12 pm](https://user-images.githubusercontent.com/15064365/41252062-0f00834a-6d8a-11e8-9ccc-f883aeb96083.png)

For context, I found this because the `Cone` Python class generated from the schema was missing a `color` property 🙂

**Comment by etpinard** [comment]
No this is correct. There's no `color` attribute for cones yet. Cones are colored by their corresponding u/v/w norm and that's all we allow at the moment.

**Comment by jmmease** [comment]
Ok, that makes sense. Thanks for the info.

I guess the less critical issue is that the default descriptions on other color-related properties still refer to a `color` property.

e.g. The description of `colorscale` reads "Sets the colorscale and only has an effect if `color` is set to a numerical array. ..."  `cmin` and `cmax` also have similar language.

Since these descriptions will now be included as Python docstrings it would be nice to get the "u/v/w norm" language in there at some point.

And the cone trace looks awesome BTW!

**Comment by etpinard** [comment]
> I guess the less critical issue is that the default descriptions on other color-related properties still refer to a color property.

Oops. Thanks for letting us know!

**Comment by CarlAndersson** [comment]
I agree with @jzthree that direct control of the color can be really useful in some cases.

**Comment by RinconFer** [comment]
Hi, It's been a long time since this issue was raised again, but I am using plotly to render UMAP embedding from scRNAseq where I use the cones direction and size to show the RNA velocity. I would like to be able to use the color variable to plot other variables (clustering, cell type, sample, latent time,...) Is there any way to change the color already? If it is still not available, is that option expected to be included soon?
Basically I would like to combine this go.Cone figure
![image](https://user-images.githubusercontent.com/62716429/228407001-5706b15d-b7c9-4e31-966f-2bfb39769ef9.png)
With the colors in this scatter_3d
![image](https://user-images.githubusercontent.com/62716429/228407177-28bb29f8-7d1c-4b31-9c83-1ecc7950e1c1.png)
